web/client/actions/geostory.js

```diff
@@ -134,6 +137,14 @@ export const setCurrentStory = (story) => ({ type: SET_CURRENT_STORY, story});
  * Turn on/off preview in cards.
 */
 export const toggleCardPreview = () => ({ type: TOGGLE_CARD_PREVIEW});
+/**
+ * Turn on/off settings options in panel, like logo etc.
+*/
+export const toggleSettings = (option) => ({ type: TOGGLE_SETTINGS, option});
```

Comment:
it should be singular (it toggles a single setting)

Reply:
ok